### PR TITLE
ISLANDORA-1919

### PR DESCRIPTION
**Pull request by jonathangreen**
**JIRA Ticket**: (link)

https://jira.duraspace.org/browse/ISLANDORA-1919

# What does this Pull Request do?

Attempt to rectify undocumented function signature change by adding default parameters to the function.

# What's new?

Function only returns values for parents thumbnails if new parameters is set to TRUE.

# Interested parties
@DiegoPino @willtp87 @jordandukart

### CODE REVIEW

**Comment by DiegoPino** [comment]
😃

File: islandora_compound_object.module [hunk 1]
```diff
@@ -560,8 +560,11 @@ function islandora_compound_object_menu_local_tasks_alter(&$data, $router_item,
  * Helper function to retrieve relevant information about the compound.
  *
  * @param AbstractObject $object
- *   An AbstractObject representing an object within Fedora. This object can
- *   either be the child of a compound, or the compound object itself.
+ *   An AbstractObject representing an object within Fedora.
+ * @param bool $allow_parent
+ *   (Optional) If this is set to true, then this functin will return
```
Comment (jordandukart) [inline islandora_compound_object.module]: Spelling?

**Comment by jordandukart** [comment]
Looking at this this afternoon

**Review by jordandukart — COMMENTED** [review]
Beyond spelling in that doc comment, all looks good.

**Comment by jordandukart** [comment]
Looks good on my end, will wait 24hrs before merging.

**Comment by DiegoPino** [comment]
@jonathangreen @jordandukart this looks good! We will need its 7.x-1.9 counterpart also! Cheers and thanks

**Comment by jonathangreen** [comment]
Made counterpart release pull request.

**Review by jordandukart — APPROVED** [review]

**Pull request merged**